node/src/test/tests.rs

```diff
@@ -167,6 +168,50 @@ fn test_inventory_sync() {
     assert_eq!(a, b);
 }
 
+#[test]
+fn test_tracking() {
+    let rng = fastrand::Rng::new();
+    let mut alice = Peer::config(
+        "alice",
+        Config {
+            project_tracking: ProjectTracking::Allowed(HashSet::with_hasher(rng.clone().into())),
+            ..Config::default()
+        },
+        [7, 7, 7, 7],
+        vec![],
+        MockStorage::empty(),
+        fastrand::Rng::new(),
+    );
+    let proj_id: identity::ProjId = test::arbitrary::gen(100);
```

Comment:
The number passed to `gen` only really matters when generating arbitrary-sized data, but since project IDs are always the same size, I'd just pass `1` as a convention. Maybe `0` works too *shrug*